en/administration/install/windows.md

```diff
@@ -1,126 +1,139 @@
-% Installing on Window
+### Prerequisites
```

Comment:
don't change the `% title` since that is used by pandoc

Reply:
Fixed, thanks.